server/authz/errors.go

```diff
@@ -52,8 +52,13 @@ func (e *Forbidden) Internal() string {
 
 // LogFields allows this error to be logged with subject, object, and action.
 func (e *Forbidden) LogFields() []interface{} {
+	// Only logging User's email, and not other details such as Password and Salt.
+	email := "nil"
+	if e.subject != nil {
+		email = e.subject.Email
+	}
 	return []interface{}{
-		"subject", e.subject,
+		"subject", email,
```

Comment:
The `kitlog.With(...` method we use cannot unmarshal structs, so using `e.subject` here always gave me `unsupported value type`. Maybe this used to work at some point when we used a different logging approach.

Reply:
Thanks for catching this!